Makefile

```diff
@@ -216,6 +219,11 @@ email.md: ChangeLog tools/email-footer.md
 blog.html: email.md
 	cat $< | ./$(NODE_EXE) tools/doc/node_modules/.bin/marked > $@
 
+doc-branch-upload: NODE_DOC_VERSION = v$(shell $(PYTHON) tools/getnodeversion.py | cut -f1,2 -d.)
```

Comment:
@bnoordhuis removed upload targets in 73b9323488a, which I'm +1 on, I'll either PR to add some new ones back or we'll keep them separate. We should keep that consistent and remove this target.